CONTRIBUTING.md

```diff
@@ -31,14 +31,17 @@ If installation fails due to a `node-gyp` issue you may need to perform some add
 
 ### on macOS
 
-- Determine if you have Xcode command line tools installed
-  - example the console will tell you if they're already installed): `xcode-select --install`
-- Determine if you have Python 2.7 installed
-  - example: `which python2.7`
-- If you do not have Python 2.7 installed, you need to install it. We recommend `pyenv` for this so you can keep the default MacOS python installation untouched.
+- Attempt to install Xcode command line tools (the console will tell you if they're already installed)
+  - example: `xcode-select --install`
+- Determine if you have Python 2.x installed
+  - example: `which python`
+- If you do not have Python 2.x installed, you need to install it. We recommend `pyenv` for this so you can keep the default MacOS python installation untouched.
```

Comment:
I don't know Python things... but does this need to be Python `2.7.x`?

Reply:
`which python` won't necessarily tell you which version is installed (for example, if you use pyenv, it'll just show you the shim).

`python --version` is probably more useful.